autogen/main/cluster.tf.tmpl

```diff
@@ -202,6 +202,10 @@ resource "google_container_cluster" "primary" {
     }
   }
 
+  service_external_ips_config {
```

Comment:
Can we make this a dynamic block if enabled?

Reply:
Sure! let me include in the changes I'll make locally